terracumber_config/tf_files/SUSEManager-4.3-build-validation.tf

```diff
@@ -815,29 +815,29 @@ module "ubuntu2004-minion" {
 
 }
 
-// module "ubuntu2204-minion" {
-//   providers = {
-//     libvirt = libvirt.mandalore
-//   }
-//   source             = "./modules/minion"
-//   base_configuration = module.base_debian.configuration
-//   product_version    = "4.3-released"
-//   name               = "min-ubuntu2204"
-//   image              = "ubuntu2204o"
-//   provider_settings = {
-//     mac                = "aa:b2:92:42:00:bb"
-//     memory             = 4096
-//   }
-//   server_configuration = {
-//     hostname = "suma-bv-43-pxy.mgr.prv.suse.net"
-//   }
-//   auto_connect_to_master  = false
-//   use_os_released_updates = false
-//   ssh_key_path            = "./salt/controller/id_rsa.pub"
-//
-//   //ubuntu2204-minion_additional_repos
-//
-// }
+module "ubuntu2204-minion" {
+  providers = {
+    libvirt = libvirt.mandalore
+  }
+  source             = "./modules/minion"
+  base_configuration = module.base_debian.configuration
+  product_version    = "4.3-released"
+  name               = "min-ubuntu2204"
+  image              = "ubuntu2204o"
+  provider_settings = {
+    mac                = "aa:b2:92:42:00:bb"
+    memory             = 4096
+  }
+  server_configuration = {
+    hostname = "suma-bv-43-pxy.mgr.prv.suse.net"
+  }
+  auto_connect_to_master  = false
+  use_os_released_updates = false
+  ssh_key_path            = "./salt/controller/id_rsa.pub"
+
+  //ubuntu2204-minion_additional_repos
```

Comment:
Context:
We had this comment in the past with the idea to support in Terracumber additional repositories for MUs directly injected during deployment.
Nowadays, we decided to do it properly, so we create custom channels inside SUMA server, and add the custom channel to the AK used during the bootstrap of the under register system.
So, I think... this comment become unuseful.

Reply:
Thank you for providing some context on this.